.github/workflows/aws.yml

```diff
@@ -0,0 +1,128 @@
+          java-version: '1.8.0'
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          # terraform_version: 0.13.0:
+          cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
+
+      - name: Terraform Format
+        working-directory: src/test/resources/terraform
+        id: fmt
+        run: terraform fmt -check
+
+      - name: Terraform Init
+        working-directory: src/test/resources/terraform
+        id: init
+        run: terraform init
+      
+      - name: Terraform Validate
+        working-directory: src/test/resources/terraform
+        id: validate
+        run: terraform validate -no-color
+
+      - name: Terraform Plan
+        working-directory: src/test/resources/terraform
+        id: plan
+        if: github.event_name == 'pull_request'
+        run: terraform plan -no-color
+        continue-on-error: true
+
+      - uses: actions/github-script@0.9.0
+        if: github.event_name == 'pull_request'
+        env:
+          PLAN: "terraform\n${{ steps.plan.outputs.stdout }}"
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const output = `#### Terraform Format and Style 🖌\`${{ steps.fmt.outcome }}\`
+            #### Terraform Initialization ⚙️\`${{ steps.init.outcome }}\`
+            #### Terraform Validation 🤖\`${{ steps.validate.outcome }}\`
+            #### Terraform Plan 📖\`${{ steps.plan.outcome }}\`
+            <details><summary>Show Plan</summary>
+            \`\`\`\n
+            ${process.env.PLAN}
+            \`\`\`
+            </details>
+            *Pusher: @${{ github.actor }}, Action: \`${{ github.event_name }}\`*`;
+            github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: output
+            })
+      - name: Terraform Plan Status
+        working-directory: src/test/resources/terraform
+        if: steps.plan.outcome == 'failure'
+        run: exit 1
+
+      - name: Terraform Apply
+        working-directory: src/test/resources/terraform
+        if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+        run: terraform apply -auto-approve
+
+      - name: Set Database Host URL
+        working-directory: src/test/resources/terraform
+        run: echo "DBHOST=$(terraform output -json | jq -r '.dbHost.value')" >> $GITHUB_ENV
+
+      - name: Create Database Objects
+        working-directory: src/test/resources/terraform
+        run: PGPASSWORD=LbRootPass1 psql -v ON_ERROR_STOP=1 --username lbuser --dbname lbcat -h $DBHOST -f postgres-init.sql
+
+  test:
+    runs-on: ubuntu-18.04
+    needs: [setup]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Update Harness Config
+        working-directory: src/test/resources
+        run: sed -i "s/DBHOST/$DBHOST/g" harness-config.yml
+
+      - name: AWS RDS Postgres Test Run
```

Comment:
@jnewton03 -- this is customized to Postgres only, right? I like the work that @mcred did to parallelize jobs for other platforms in the same workflow yaml file. Could we eventually include other (at least other AWS cloud) platforms in the same yaml via a parallel/matrix strategy?

Reply:
@kristyldatical Yes.  I'll do that in the subsequent tickets for aurora and oracle.